SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
```

Comment:
IIUC, here we briefly describe the correct way of reporting vulnerabilities. Is the Ethereum bounty program the correct way of reporting security vulnerabilities (see https://docs.google.com/forms/d/e/1FAIpQLSegLOqYnWv4hWM4RKjDiHIzhZQMavRrKllPJzYJYIPntAim7g/viewform?c=0&w=1)

We probably also need to scope out a vulnerability: This means what is in scope and what qualifies a bug as a security vulnerability?

Reply:
Yes, tell them about the bug bounty programme. In general, I would say that rules of the bug bounty programme have precedence and this here is only to have a more detailed technical discussion.